src/main/java/frc/robot/subsystems/climber/ClimberIO.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.climber;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+
+/** Add your docs here. */
+public interface ClimberIO {
+  @AutoLog
+  public static class ClimberIOInputs {
+    public double climberVelocityRotationsPerSec = 0.0;
+    public double climberAppliedVolts = 0.0;
+    public double climberCurrentAmps = 0.0;
+    public double climberTempC = 0.0;
+    public Rotation2d climberRotation = new Rotation2d();
```

Comment:
this should just be a double bc of spools being annoying to account for. should represent total rotation of the spool